invokeai/app/services/model_install/model_install_default.py

```diff
@@ -541,13 +541,10 @@ def _sync_model_path(self, key: str) -> AnyModelConfig:
         old_path = Path(model.path)
         models_dir = self.app_config.models_path
 
-        try:
-            old_path.relative_to(models_dir)
+        if not old_path.is_relative_to(models_dir):
```

Comment:
I had run into a problem with this check: https://github.com/invoke-ai/InvokeAI/pull/5911/commits/51f2954b802cda16347f9d7e651deaa74cd63e01

But in testing, I cannot reproduce it now. I tried changing the base of a safetensors model in the models dir and one outside it (installed with inplace). Maybe another change resolved the issue.

Reply:
Ok. I'll double check.

Reply:
I couldn't find a problem with the current way of checking relative addressing, but I've added a resolve() to both the `old_path` and `models_dir` in case symbolic links and relative addressing are causing problems. 

I don't understand the logic of your `try` block. We want to return immediately with the path unchanged in the event that the old path is _not_ a subpath of `models_dir`. The `try` block returns in the event that the old path _is_ a subpath, because the ValueError exception is raised when calling `relative_to()` with disjunct paths.

The motivation is that the model manager only changes paths for files/directories that are subpaths of `models_dir`

Reply:
Ugh, yeah my logic was backwards in hindsight. I can't remember why I needed to adjust the logic, but clearly I didn't do it correctly.